dsgrid/utils/spark.py

```diff
@@ -328,6 +331,9 @@ def write_dataframe_and_auto_partition(df, filename, partition_size_mb=128, colu
         Target size in MB for each partition
     columns : None, list
         If not None and repartitioning is needed, partition on these columns.
+    rtol_pct : int
+        Don't repartition or coalesce if the relative difference between target and desired
```

Comment:
I think target and desired mean the same thing here?

Reply:
Yeah, typo